include/n64dd.h

```diff
@@ -2,39 +2,37 @@
 #define N64DD_H
 
 #include "ultra64.h"
+#include "z64pause.h"
 #include "z64scene.h"
 
 struct GameState;
 union Gfx;
 struct PlayState;
-struct PauseMapMarksData;
 struct RegEditor;
 struct RoomContext;
 struct SaveContext;
 struct Scene;
 
 // TODO Use the specific pointer types instead of void*
-typedef struct n64ddStruct_800FF4B0_pointers {
+typedef struct n64ddStruct_800FEE70_pointers {
     void* unk_00;
     void* unk_04;
     struct RegEditor* unk_08;
     void* unk_0C[31];
     struct SaveContext* unk_88;
     void* unk_8C[9];
-} n64ddStruct_800FF4B0_pointers; // size = 0xB0
+} n64ddStruct_800FEE70_pointers; // size = 0xB0
 
-struct n64ddStruct_80121AF0;
-
-typedef struct n64ddStruct_80121AF0 {
-    void (*unk_00)(n64ddStruct_800FF4B0_pointers*, struct n64ddStruct_80121AF0*);
+typedef struct n64ddStruct_80121220 {
+    void (*unk_00)(n64ddStruct_800FEE70_pointers*, struct n64ddStruct_80121220*);
     void (*unk_04)(void);
     void (*unk_08)(struct PlayState* play, struct RoomContext* roomCtx, s32 roomNum);
     void (*unk_0C)(struct PlayState* play);
     void (*unk_10)(struct PlayState* play);
     void (*unk_14)(struct PlayState* play);
     char unk_18[0x1C];
-    void (*unk_34)(struct PauseMapMarksData**);
-    void (*unk_38)(struct PauseMapMarksData**);
+    void (*unk_34)(PauseMapMarksData**);
```

Comment:
why not keep the forward reference

Reply:
this is fixing a mistake, `PauseMapMarksData` is not a struct, it's a typedef for an array, and CC_CHECK errors. I broke this in #2117 and fixed it in #2109

Reply:
(I've merged main so this diff no longer appears in this PR)